lib/internal/blob.js

```diff
@@ -78,6 +78,7 @@ let ReadableStream;
 let URL;
 
 const enc = new TextEncoder();
+let dec;
```

Comment:
Initialize decoder just like Encoder?
```suggestion
const dec = new TextDecoder();
```

Reply:
https://github.com/nodejs/node/issues/44788